resources/bluima_sentence_detector_genia/src/main/java/ch/epfl/bbp/nlp/res/sentence/GeniaResource.java

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.bbp.nlp.res.sentence;
+
+import ch.epfl.bbp.nlp.ModelResource;
+
+/**
+ * Genia model for sentence splitter annotator
+ */
+public class GeniaResource implements ModelResource {
+    public String getResourcePath() {
+        return "/SentDetectGenia.bin.gz";
```

Comment:
I've no strong position on this one. The only minor thing with it would be the duplicate information (pom + filename). So let me know what you prefer and I'll do the change.